scene/resources/packed_scene.cpp

```diff
@@ -1054,7 +1055,14 @@ Error SceneState::_parse_connections(Node *p_owner, Node *p_node, HashMap<String
 		for (const Node::Connection &F : conns) {
 			const Node::Connection &c = F;
 
-			if (!(c.flags & CONNECT_PERSIST)) { //only persistent connections get saved
+			// Don't save connections that are not persistent
+			if (!(c.flags & CONNECT_PERSIST)) {
```

Comment:
```suggestion
			// Don't save connections that are not persistent.
			if (!bool(c.flags & CONNECT_PERSIST)) {
```
Consistent with the one below.